config/application.rb

```diff
@@ -1,6 +1,21 @@
 require_relative 'boot'
 
-require 'rails/all'
+# Include each railties manually, excluding `active_storage` and `action_cable`
```

Comment:
I think it is enough to specify list of requires:

```ruby
require 'active_record/railtie'
require 'action_controller/railtie'
...
```

the same rails does

https://github.com/rails/rails/blob/4a835aa3236eedb135ccf8b59ed3c03e040b8b01/railties/lib/rails/generators/rails/app/templates/config/application.rb#L3-L18

Reply:
done :+1: